docgen/src/guides/using-multiple-instantsearch.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Using multiple InstantSearch instances
+layout: guide.pug
+category: guide
+navWeight: 0
+---
+
+Having several instances of `InstantSearch` can be useful when: 
+
+* You want to make a search on multiple indices 
+* You want to synchronize multiple `InstantSearch` instances
```

Comment:
"Having several instances of `InstantSearch` can be useful" => "You want to synchronize multiple `InstantSearch` instances" is not easy to understand.

I would rather say: "
You can use multiple `<InstantSearch/>` instances for cases like:
- displaying hits from different indices,
- while sharing a single SearchBox,
- or any use case involving synchronizing widgets between different `<InstantSearch>` instances
"